static/less/style.less

```diff
@@ -194,20 +194,66 @@ img {
         margin-left: 1.0rem;
         margin-right: 1.0rem;
       }
+      @media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+        float: none !important;
+      }
+    }
+    @media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+      visibility: hidden;
+      opacity: 0;
+      max-height: 0;
+      z-index: 5;
+      top: 5rem;
+      right: 0;
+      width: 100%;
+      position: absolute;
+      background-color: rgba(254,254,254 ,0.98);
+      padding: 0;
+      border-bottom: solid 2px #E2DFE1;
+      transition: opacity 0.25s, max-height 0.15s linear;
     }
   }
 }
 
-@media only screen and (min-device-width : 320px) and (max-device-width : 480px) {
-  a.navigation-title {
-    font-size: 0rem;
+#menu-control {
+  display: none;
+}
 
-    &::after {
-      content: '~';
-      font-size: 2.4rem;
-      text-align: center;
-      margin-left: -1.4rem;
-    }
+.btn-mobile {
+  display: none;
+}
+
+@media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+  .btn-mobile {
+    display: block;
+    font-size: 2.5rem;
+    color: black;
+    cursor: pointer;
+    padding-top: 1.5rem;
+  }
+
+  #menu-control:checked + label .btn-mobile {
+    color: #E2DFE1;
+  }
+
+  #menu-control:checked + label ul {
+    visibility: visible;
+    opacity: 1;
+    max-height: 100rem;
+  }
+
+  .navigation-item {
+    text-align: center;
+    position: relative;
+  }
+
+  .mobile-menu-lang-separator {
+    padding-left: 7rem;
```

Comment:
I personally like these to be `1.5rem` (both left and right) but I'm generally ok with this and specially that CSS can be easily overriden.

Reply:
ahahaha
I prefer a smaller line. But I can leave it as is now.

With the new SCSS pipeline it will be easier to override any style rule.